doc/man1/openssl-verify.pod

```diff
@@ -329,11 +329,11 @@ required by the SM2 signature algorithm for signing and verification.
 Specify a binary ID string to use when signing or verifying using an SM2
 certificate. The argument for this option is string of hexadecimal digits.
 
-=item B<->
+=item B<-->
 
 Indicates the last option. All arguments following this are assumed to be
 certificate files. This is useful if the first certificate filename begins
-with a B<->.
+with a B<-->.
```

Comment:
Making this a double-minus seems wrong.

Reply:
It aligns with your code.

https://github.com/openssl/openssl/blob/master/apps/lib/opt.c#L647-L650

Reply:
Yes, but your rationale is wrong, you want to end the options with ```--``` in case a filename starts with ```-``` so that ```-verify.pem``` is not mistaken for a flag.

Reply:
Oh, sorry, misread.  Okie, will fix